pkgs/by-name/zb/zbus-xmlgen/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchCrate, makeBinaryWrapper, rustfmt }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "zbus_xmlgen";
+  version = "3.1.1";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-vaefyfasOLFFYWPjSJFgjIFkvnRiJVe/GLYUQxUYlt0=";
+  };
+
+  cargoSha256 = "WXJ49X4B2aNy1zPbTllIzRhZJvF+RwfQ0Hhm/D+LQfk=";
```

Comment:
Use `cargoHash` instead as described in https://github.com/NixOS/nixpkgs/blob/965120bec609e6d5508a378028e04f35f3edcdc9/doc/languages-frameworks/rust.section.md?plain=1#L50